app/views/shared/_site_sidebar.html.erb

```diff
@@ -1,6 +1,8 @@
-<%= render 'search_across/search_form', presenter: Blacklight::SearchBarPresenter.new(controller, SearchAcrossController.blacklight_config) %>
-<% if defined? @response %>
-  <%= render 'catalog/search_sidebar' %>
+<% if Settings.feature_flags.search_across %>
```

Comment:
I think we'll want to show the search form on the search across pages even if the feature flag is false?

Reply:
Got it.. I think this change gets at what we want?